docs/manual/guide/build/code/lagom-build.sbt

```diff
@@ -1,5 +1,5 @@
 //#add-sbt-plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.0.0-M1")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "X.Y.Z") // replace 'X.Y.Z' with your preferred version (e.g. '1.2.0-RC2').
```

Comment:
It would be good if we had a way to inject the latest released version of the current branch into this somehow... I don't know if there's an easy solution, but it's something to think about.